packages/presentation/lib/src/bloc/events.dart

```diff
@@ -0,0 +1,11 @@
+abstract class GameEvent {}
+
+class MakeAttempt extends GameEvent {}
+
+class NewGame extends GameEvent {}
+
+class SetSuggestedNumber extends GameEvent {
+  final int number;
```

Comment:
Small clarification: Maybe it'll better to put word "Event" to Event name in the end